presto-main/src/main/java/com/facebook/presto/server/TaskResource.java

```diff
@@ -324,6 +324,17 @@ public void abortResults(@PathParam("taskId") TaskId taskId, @PathParam("bufferI
         taskManager.abortTaskResults(taskId, bufferId);
     }
 
+    @DELETE
+    @Path("{taskId}/remote-source/{sourceTaskId}")
```

Comment:
We use camel case for `Path`: `remoteSource`. (e.g., `ClusterStatsResource::getWorkerMemoryInfo`)